.github/workflows/build.yml

```diff
@@ -44,11 +60,16 @@ jobs:
             #export CXX=clang++
```

Comment:
We can remove this, can we not?

Reply:
I treat that as a comment note that `clang++` is the default way to do it, since we're using `clazy` instead.